### PR TITLE
Add Divy as a regular member

### PR DESCRIPTION
**Pull request by designMoreWeb**
I have been active for the past 3 months with Node website redesign as well as have been attending the CPC meetings regularly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Anyone who has been active in the foundation or one of its member projects, as d
 - Sendil Kumar (@sendilkumarn)
 - Tierney Cyren (@bnb)
 - Waleed Ashraf (@waleedashraf)
-
+- Divy Tolia (@designmoreweb)
```
Comment (joesepi) [inline README.md:110]: Please alphabetize. Thanks!

Comment (designMoreWeb) [reply README.md]: fixed

**Comment by joesepi** [comment]
Here we go again @designMoreWeb -- you've made a lot of unnecessary changes, I assume because of some built in "fixing" that your editor is doing. Please fix the fixes. ;)

**Comment by designMoreWeb** [comment]
Need to add 'cross-project-council-agenda' label

**Pull request merged**